stratum/hal/lib/barefoot/bfrt_node_test.cc

```diff
@@ -193,68 +207,34 @@ class BfrtNodeTest : public ::testing::Test {
   static constexpr int kLogicalPortId = 35;
   static constexpr uint32 kPortId = 941;
   static constexpr uint32 kL2McastGroupId = 20;
-  static constexpr char kBfConfigPipelineString[] = R"PROTO(
+  static constexpr char kBfConfigPipelineString[] = R"pb(
```

Comment:
This change comes from a newer version of clang-format (I think 12 or 13). For now we can leave it at `PROTO`.